search-api/config.py

```diff
@@ -10,7 +10,20 @@
 BASE_DIR = os.path.abspath(os.path.dirname(__file__))
 
 # # Example DB Configuration
-SQLALCHEMY_DATABASE_URI = f"sqlite:////{BASE_DIR}/db/db.sqlite3"
+
+
+if os.environ.get("POSTGRES_DB_HOST", False):
```

Comment:
**Super Small Comment**

I think we should use None instead of False here, because `False` actually is a value, unlike `None`.